.github/workflows/pr_verification.yml

```diff
@@ -82,7 +82,15 @@ jobs:
       - name: Run unit tests
         uses: gradle/gradle-build-action@v2
         with:
-          arguments: test
+          arguments: test coverage
+
+      - name: Archive reports
+        uses: actions/upload-artifact@v3
+        with:
+          name: jacocoReports
+          path: |
+            **/build/test-results/**/*.xml
+            **/build/jacoco/test.exec
```

Comment:
I don't think we should be filtering anything here. The filtering should only happen on sonarcloud

Reply:
nevermind, this isn't exclusions.